src/utils/hex.cpp

```diff
@@ -71,7 +71,7 @@ int Hex::toInt(char c) {
 std::string Hex::forRPC() const {
   std::string retHex = this->hex;
   if (retHex[0] != '0' && retHex[1] != 'x') retHex.insert(0, "0x");
-  if (retHex == "0x") { retHex = "0x0"; return retHex; };
+  if (retHex == "0x") { return retHex; };
```

Comment:
Isn't RPC supposed to be "0x0" here as per Ethereum standards (check the header comment on this function)? If yes please fix, if not please ignore.